CHANGELOG.md

```diff
@@ -1,13 +1,39 @@
 # Change Log
 
-## [1.7.0](https://github.com/chef/chef-provisioning-aws/tree/1.7.0) (2015-11-19)
-[Full Changelog](https://github.com/chef/chef-provisioning-aws/compare/v1.6.1...1.7.0)
+## [Unreleased](https://github.com/chef/chef-provisioning-aws/tree/HEAD)
```

Comment:
Yep. We need to figure out how to make the rake task do this without screwing up when you run in between releases :/